core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -83,13 +83,15 @@
 
         var showDropdown = function() {
           var togglePosition = toggle.offset();
+          var $notebook = $('bk-notebook');
+          var notebookPosition = $notebook.offset();
```

Comment:
Yup on user interaction which means other things might also be happening, sometimes in this app as you have seen it will randomly feel sluggish on user interactions. Ideally we would guard against these perf pitfalls by coding in these solutions to our directives. 

But yeah it is only "2" recalcs of the entire document layout, which in isolation with nothing else happening would most likely not hurt FPS in a noticeable way. Given how complex this app is, I would not bet on that fact.